versatile-api/src/main/java/com/fb/versatile_api/Hotel.java

```diff
@@ -0,0 +1,54 @@
+package com.fb.versatile_api;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class Hotel {
+    private final String hotelName = "Black Forest Hotel";
```

Comment:
Here you have initialized all the class fields except `rooms` while defining them. That means this class can't be used to create a new Hotel, It's always fixed to `Balck Forset Hotel`.

As a developer and best practices, we should always separate the class definition from object initialization even if we use it for one object.